packages/client-sdk-web/src/internal/cache-data-client.ts

```diff
@@ -988,22 +989,19 @@ export class CacheDataClient<
       );
     }
     this.logger.trace(`Issuing 'getBatch' request; keys: ${keys.toString()}`);
```

Comment:
For a rainy day, we should use format strings for the trace log statements instead of interpolating.

Reply:
yeah actually I just want to get rid of these altogether, and use middleware for per-request logging.